saleor/graphql/app/mutations/app_install.py

```diff
@@ -63,6 +65,16 @@ def clean_input(cls, info, instance, data, **kwargs):
         manifest_url = data.get("manifest_url")
         clean_manifest_url(manifest_url)
 
+        if app := App.objects.filter(manifest_url=manifest_url).first():
```

Comment:
You can have the same app deployed in different places. All of them will result in the same `app_identifier`. I don't think that we should rely on the uniqueness of `manifest_url`. It should be uniqness for `app.identifier`

Reply:
Additionally: You can have local env, when you use the same URL localhost for serving different app's manifest (apps with different identifiers). This will cause, you will not be able to install them.

Reply:
@Air-t  lets also change the title of the PR to match to the modified content :)